client/.prettierignore

```diff
@@ -2,4 +2,6 @@ node_modules
 README.md
 .eslintrc.json
 package-lock.json
-package.json
\ No newline at end of file
+package.json
```

Comment:
We do not want to be adding anything to the ignore as the jsx files should always be checked for style and linting